data/core/units/saurians/Javelineer.cfg

```diff
@@ -0,0 +1,98 @@
+#textdomain wesnoth-units
+[unit_type]
+    id=Saurian Javelineer
+    name= _ "female^Saurian Javelineer"
+    race=lizard
+    gender=female
+    image="units/saurians/javelineer/javelineer.png"
+    profile="portraits/saurians/skirmisher.webp"
+    hitpoints=50
+    movement_type=lizard
+    movement=7
+    experience=150
+    level=3
+    alignment=chaotic
+    advances_to=null
+    {AMLA_DEFAULT}
+    cost=47
+    usage=archer
+    description= _ "After years of combat practice, saurians warriors that are the most talented in the skill of spear throwing might advance to the ranks of the javelineers. With better equipment and exceptional aim, saurian javelineers are not to be underestimated."
+    die_sound=hiss-die.wav
+    [attack]
+        name=spear
+        description=_"spear"
+        range=melee
+        type=pierce
+        damage=8
```

Comment:
I think this unit should focus even more on ranged and less melee. So weaker melee, stronger ranged, say -1 melee dmg and +1 ranged dmg, or however you want to do it.

Reply:
@nemaara I think 8x3 melee attack draws an interesting parallel to the Flanker, as it deals the same damage per hit with 1 more strike. I can lower it to 7x3, but I think overall it's fitting for a unit that starts with a melee archetype.

As for the ranged damage, I agree it can be raised - especially if the marksman ability should be removed. If it was up to me, I'd probably leave melee damage as is and increase the ranged damage accordingly to 14 or 15, but it's your call.

Reply:
Okay that sounds fine to me but maybe @Hejnewar would also want to weigh in.

Reply:
I've changed the attack to 14x3 and removed the marksman ability. Let me know if that works for you.

Reply:
I'm trying to catch up, I believe I missed something honestly but okay, this seems okay, usually I simply don't do too much unless someone would try to do something too broken and then I stop that. :P I dont have any strong feelings about this unit so I just leave it for others. 

Actually I will mention one thing: saurians don't seem like the types to throw this strong, I think having more strikes and less damage is more friendly honestly for agility based species.

Reply:
I agree that it fits thematically. I decided to put only a few attacks for consistency with the original skirmisher tree. However, it makes sense for a ranged-focused unit to have even more throws. Current damage of 42 (14x3) is not a multiple of 4 or 5, so I'm not exactly sure what values to choose to maintain a similar power level. Do you think 10x4 is OK, or should I go with another set of values? @Hejnewar @nemaara

Reply:
9-5 will land this unit pretty much in the middle of all the level 3 units when it comes to strength (and will also raise the price to 52), in comparison Flanker is second to last when it comes strength.

Reply:
I've adjusted the damage and cost, let me know if the units stats are acceptable now.

Reply:
Yes everything is fine.

Reply:
@nemaara Can you review the stats again?